esp/logging/loggingagent/CMakeLists.txt

```diff
@@ -14,3 +14,4 @@
 #    limitations under the License.
 ################################################################################
 HPCC_ADD_SUBDIRECTORY (espserverloggingagent)
+HPCC_ADD_SUBDIRECTORY (cassandraloggingagent)
```

Comment:
I'm just wondering if the structure could be more explicit about the relationship between components. Can we consider cassandraloggingagent to be a plug-in ? if so could we have a logginagent parent dir, and a plugins folder under it, which contains cassandralogging and whatever else we add in the future?